unitary/alpha/qudit_gates.py

```diff
@@ -32,3 +32,59 @@ def _unitary_(self):
 
     def _circuit_diagram_info_(self, args):
         return f"[+{self.addend}]"
+
+
+class QuditSwapPowGate(cirq.Gate):
+    def __init__(self, dimension: int, exponent: int = 1):
+        self.dimension = dimension
+        self.exponent = exponent
+
+    def _qid_shape_(self):
+        return (self.dimension, self.dimension)
+
+    def _unitary_(self):
```

Comment:
The matrix is not unitary. Some columns are all zero

Reply:
Thanks.  I was worried this matrix was not correct.  I will take another stab at it tomorrow.

Reply:
Partially fixed these gates and added a test to check unitarity for the gates.  iSwap is still not quite right though.  (and I don't have high confidence in swap yet either).  One step closer though.